drivers/hud_data_quality_report/app/models/hud_data_quality_report/document_exports/hud_data_quality_report_export.rb

```diff
@@ -7,60 +7,21 @@
 module HudDataQualityReport::DocumentExports
   class HudDataQualityReportExport < ::GrdaWarehouse::DocumentExport
     include ApplicationHelper
-    def authorized?
-      return true if user.can_view_all_hud_reports?
-
-      user.can_view_hud_reports? && report.present?
-    end
+    include HudReports::HudPdfExportConcern
 
     def generator_url
-      hud_reports_dq_path(report)
-    end
-
-    private def report_scope
-      scope = report_class.
-        where(report_name: report_generator_class.title)
-      return scope if user.can_view_all_hud_reports?
-
-      scope.where(user_id: user.id)
-    end
-
-    protected def report
-      @report ||= report_scope.find_by(id: params['id'])
-    end
-
-    protected def view_assigns
-      {
-        report: report,
-        generator: report_generator_class,
-      }
-    end
-
-    private def report_generator_class
-      HudDataQualityReport::Generators::Fy2020::Generator
-    end
-
-    def perform
-      with_status_progression do
-        template_file = 'hud_reports/download'
-        PdfGenerator.new.perform(
-          html: view.render(file: template_file, layout: 'layouts/hud_report_export'),
-          file_name: "DQ 2020 #{DateTime.current.to_s(:db)}",
-        ) do |io|
-          self.pdf_file = io
-        end
-      end
+      hud_reports_spm_path(report)
```

Comment:
I think this should be `hud_reports_dq_path(report)`

Reply:
Thanks.  I knew I missed at least one.